src/core/CONVENTIONS.md

```diff
@@ -1,15 +1,33 @@
-# Kibana Conventions
-
-- [Kibana Conventions](#kibana-conventions)
+- [Organisational Conventions](#organisational-conventions)
+  - [Definition of done](#definition-of-done)
+- [Technical Conventions](#technical-conventions)
   - [Plugin Structure](#plugin-structure)
     - [The PluginInitializer](#the-plugininitializer)
     - [The Plugin class](#the-plugin-class)
     - [Applications](#applications)
     - [Services](#services)
     - [Usage Collection](#usage-collection)
     - [Saved Objects Types](#saved-objects-types)
-
-## Plugin Structure
+  - [Naming conventions](#naming-conventions)
+
+## Organisational Conventions
+### Definition of done
+Definition of done for a feature:
+- has a dedicated Github issue describing problem space
+- an umbrella task closed/updated with follow-ups
+- all code review comments are resolved
+- works for OSS version
+- works for the commercial version with a license
+   - for a logged-in user
+   - for anonymous user
+   - compatible with Spaces
```

Comment:
- works with and without a `server.basePath` configured
- cannot crash the Kibana server when it fails

Reply:
> cannot crash the Kibana server when it fails

Fails in what case? Kibana crashes if a plugin throws an exception in setup or start lifecycle.

Reply:
I think Josh was probably thinking of the criteria in the Status RFC https://github.com/elastic/kibana/pull/59621 We should be sure that exceptions would only bubble up to the `fatal` handler if it's truly impossible to resolve them.

Reply:
> Fails in what case? Kibana crashes if a plugin throws an exception in setup or start lifecycle.

I left this intentionally vague since each change is different. Basically, we should try to imagine ways to crash the server and make sure they can't happen.

Reply:
ok, added as is